### PR TITLE
[Merged by Bors] - feat: Int.odd_pow

### PR DESCRIPTION
**Pull request by mo271**
analogous to `Int.even_pow` and `Int.even_pow'`

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by eric-wieser** [comment]
Please run `git rebase -i origin/master` and remove the commits that are not yours, then `git push origin --force-with-lease`.

**Comment by mo271** [comment]
> Please run `git rebase -i origin/master` and remove the commits that are not yours, then `git push origin --force-with-lease`.

done

File: Mathlib/Data/Int/Parity.lean [hunk 1]
```diff
@@ -164,6 +164,13 @@ theorem even_pow' {n : ℕ} (h : n ≠ 0) : Even (m ^ n) ↔ Even m :=
   even_pow.trans <| and_iff_left h
 #align int.even_pow' Int.even_pow'
 
+@[parity_simps]
+theorem odd_pow {n : ℕ} : Odd (m ^ n) ↔ Odd m ∨ n = 0 := by
+  rw [← not_iff_not, ← Int.even_iff_not_odd, not_or, ← Int.even_iff_not_odd, Int.even_pow]
+
+theorem odd_pow' {n : ℕ} (h : n ≠ 0) : Odd (m ^ n) ↔ Odd m := by
+  rw [← not_iff_not, ← Int.even_iff_not_odd, ← Int.even_iff_not_odd, Int.even_pow' h]
```
Comment (Ruben-VandeVelde) [inline Mathlib/Data/Int/Parity.lean:172]: Could also be
```suggestion
  odd_pow.trans <| or_iff_left h
```

**Comment by eric-wieser** [comment]
Surely the analogy is to `even` not `add`?

File: Mathlib/Data/Int/Parity.lean (hunk 1 above)
Comment (mo271) [reply Mathlib/Data/Int/Parity.lean]: sure

**Comment by mo271** [comment]
> Surely the analogy is to `even` not `add`?

Yeah, that's what I meant. Will edit...

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**